crowdstrikeoauthapi_connector.py

```diff
@@ -1668,6 +1681,74 @@ def _handle_list_detections(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def _handle_get_detections_details(self, param):
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        ids = [det_id.strip() for det_id in param.get("detection_ids").split(',')]
+        ids = list(set(filter(None, ids)))
+        if not ids:
+            return action_result.set_status(phantom.APP_ERROR, CROWDSTRIKE_ERROR_INVALID_ACTION_PARAM.format(key="ids"))
+
+        data = {"ids": ids}
+
+        detection_details_list = self._get_details(action_result, CROWDSTRIKE_LIST_DETECTIONS_DETAILS_ENDPOINT, data, method='post')
+
+        if detection_details_list is None:
+            return action_result.get_status()
+
+        [action_result.add_data(detection) for detection in detection_details_list]
+
+        summary = action_result.update_summary({})
+        summary['total_detections'] = len(detection_details_list)
+
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def _handle_update_detections(self, param):
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        assigned_to_user = param.get('assigned_to_user')
+        comment = param.get('comment')
+        show_in_ui = param.get('show_in_ui', True)
+        status = param.get('status')
+        ids = [det_id.strip() for det_id in param.get("detection_ids").split(',')]
+        ids = list(set(filter(None, ids)))
+        if not ids:
+            return action_result.set_status(phantom.APP_ERROR, CROWDSTRIKE_ERROR_INVALID_ACTION_PARAM.format(key="ids"))
+
+        data = {
+            'ids': ids,
+            'show_in_ui': show_in_ui
+        }
+
+        if assigned_to_user:
+            data.update({'assigned_to_uuid': assigned_to_user})
```

Comment:
```suggestion
            data['assigned_to_uuid] = assigned_to_user
```
No need to update for a single key, same below